kolibri/plugins/learn/assets/src/views/exam-report-viewer.vue

```diff
@@ -79,6 +82,9 @@
         closed: state => state.pageState.examLog.closed,
       },
     },
+    $trs: {
+      reportsNotAvailableForExam: 'Reports are not available for this exam',
```

Comment:
Not sure what the alternative is, but we should not be introducing new strings in a patch release.

@indirectlylit @jtamiace any creative ideas for not adding strings in this PR?

Reply:
My recommendation would be to put a big, centered [Material 'X'](https://material.io/tools/icons/?search=close&icon=close&style=baseline) in the place of all these strings for 0.10.x:

![image](https://user-images.githubusercontent.com/2367265/42841776-04431236-89c0-11e8-9976-8be95212d338.png)

Then switch that over to the strings in this PR in 0.11.x